build.gradle

```diff
@@ -500,6 +500,14 @@ project('spring-integration-mail') {
 	[compileJava,compileTestJava]*.options*.compilerArgs = ["${xLintArg},-path"]
 }
 
+project('spring-integration-mock') {
```

Comment:
We'll need to think about the package name too.

`org.springframework.integration.testing`, `org.springframework.integration.testframework` ?